### PR TITLE
LPS-142097 Apply style to modal

### PR DESCRIPTION
**Pull request by naraandrade**

### CODE REVIEW

**Comment by naraandrade** [comment]
ci:test:relevant

File: modules/apps/portal-workflow/portal-workflow-taglib/src/main/resources/META-INF/resources/workflow_status/css/main.scss [hunk 1]
```diff
@@ -0,0 +1,20 @@
+.modal-full-screen {
+	display: flex;
+	justify-content: center;
+
+	.modal-content,
+	.modal-body {
+		width: 1050px;
```
Comment (ramonmsp) [inline modules/apps/portal-workflow/portal-workflow-taglib/src/main/resources/META-INF/resources/workflow_status/css/main.scss:7]: Maybe we should set different width/height depending on screen size. Modal gonna be responsible and will prevent bad behaviors.
What do you think?

**Comment by naraandrade** [comment]
ci:test:sf

**Comment by naraandrade** [comment]
ci:test:relevant

**Review by rafaprax — APPROVED** [review]

**Review by ramonmsp — APPROVED** [review]

**Comment by ramonmsp** [comment]
ci:test:relevant

**Review by jotascript — APPROVED** [review]

**Comment by anabuchmann** [comment]
Just started reviewing :)

**Comment by anabuchmann** [comment]
Manually forwarded here: https://github.com/brianchandotcom/liferay-portal/pull/109700

**Pull request closed without merging**